.github/workflows/bench.yml

```diff
@@ -45,3 +46,38 @@ jobs:
           store_results: ${{ github.repository_owner == 'pq-code-package' && github.ref == 'refs/heads/main' }}
           bench_extra_args: ${{ matrix.target.bench_extra_args }}
           gh_token: ${{ secrets.AWS_GITHUB_TOKEN }}
+  bench_ec2:
+    name: ${{ matrix.target.name }} ${{ matrix.opt.name }}
+    permissions:
+      contents: 'read'
+      id-token: 'write'
+    strategy:
+      fail-fast: false
+      matrix:
+        opt:
+          - name: opt
+            value: true
+          - name: non-opt
+            value: false
+        target:
+          - name: Graviton2
+            ec2_instance_type: t4g.small
+            ec2_ami: ubuntu-latest (aarch64)
+            archflags: -mcpu=cortex-a76 -march=armv8.2-a
+            cflags: -DFORCE_AARCH64
+          - name: Graviton3
+            ec2_instance_type: c7g.medium
+            ec2_ami: ubuntu-latest (aarch64)
+            archflags: -mcpu=neoverse-v1 -march=armv8.4-a
+            cflags: -DFORCE_AARCH64
+    uses: ./.github/workflows/bench_ec2_reusable.yml
+    if: github.repository_owner == 'pq-code-package' && (github.event.label.name == 'benchmark' || github.ref == 'refs/heads/main')
+    with:
+      ec2_instance_type: ${{ matrix.target.ec2_instance_type }}
+      ec2_ami: ${{ matrix.target.ec2_ami }}
+      archflags: ${{ matrix.target.archflags }}
+      cflags: ${{ matrix.target.cflags }}
+      opt: ${{ matrix.opt.value }}
+      store_results: ${{ matrix.opt.value }} # Only store optimized results
```

Comment:
store results should only be true if it's the main branch, right?

Reply:
This is related to https://github.com/pq-code-package/mlkem-c-aarch64/pull/126/files

It was due to that we didn't disambiguate opt vs. non-opt in the stats pushed to github pages.

Reply:
that's okay, but it should still only store it on the main branch
currently it creates a lot of commit for G2 and G3 on the PRs - that is inconsistent with the other benchmarks.

Reply:
oh yes! your're right

Reply:
@potsrevennil Can you open a separate PR for this?

Reply:
ok